tests/queries/0_stateless/02966_cbor_format.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+
+$CLICKHOUSE_CLIENT --query="DROP TABLE IF EXISTS cbor";
+
+$CLICKHOUSE_CLIENT --query="CREATE TABLE cbor (uint8 UInt8, uint16 UInt16, uint32 UInt32, uint64 UInt64, int8 Int8, int16 Int16, int32 Int32, int64 Int64, float Float32, double Float64, string String, date Date) ENGINE = Memory";
```

Comment:
Consider using .sql tests instead. It will make things simpler and allow the fuzzer understand these queries and run more of them.

Reply:
Also, it needs way more tests for both IN and OUT outputs